stdpopsim/catalog/arabidopsis_thaliana.py

```diff
@@ -118,11 +118,11 @@ def _sma_1pop():
             description="Arabidopsis Thaliana South Middle Atlas population")
     ]
 
-    return stdpopsim.Model(
+    return stdpopsim.DemographicModel(
         id="SMA_1pop",
         name="South Middle Atlas population size history",
         description="""
-            Model estimated from two homozygous individuals from
+            DemographicModel estimated from two homozygous individuals from
```

Comment:
This is a bit overzealous from sed --- I think the original English description is better.

Reply:
Ditto for various replacements of the word "Model" in English sentences below.